catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/transformer/WorkspaceKeyTransformation.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard.workspace.transformer;
+
+/**
+ * This partial implementation of {@link WorkspaceTransformation} contains default implementations
+ * of value transformation related interface methods that accept any Objects and return the same
+ * values. Implementations of this interface are therefore only able to change the key between a
+ * metacard attribute and a JSON-style key-value pair; the value remains consistent.
+ */
```

Comment:
Should probably drop the experimental blurb in here. Is this an internal interface? If so we should also put it into a .internal package